src/Provider.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
It'll be better to change the license to GPL.

Reply:
Resolved.